client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.component.js

```diff
@@ -0,0 +1,6 @@
+import template from './cdn-dedicated-manage-statistics.html';
+
+angular.module('App').component('cdnStatistics', {
+  template,
+  controller: 'CdnStatisticsCtrl',
```

Comment:
Why not `export` class and just `import controller from './controller` like you did for `template` ?

Reply:
Yes I could've do it, but this implies to create a `index.js` (placed in grand-parent directory) that list all `import controller from './controller'`, and with this file created, this also implies to refactor parent controllers etc etc...

So I've just prepared the path to `export`/`import`, (with the creation of the angularjs controller placed at the bottom of https://github.com/ovh-ux/ovh-manager-dedicated/pull/1006/files/448ad3d4d0c3696fa60ecc7ace153f6c20628010#diff-8b8547fec71a561e51a97106c23b8f27 file)